### PR TITLE
fix(css): new pages for <url> and url()

### LINKED ISSUE #35029: New <url> and src() pages
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/url

### What specific section or headline is this issue about?

This page

### What information was incorrect, unhelpful, or incomplete?

This page conflates `<url>` with `url()`. Many pages that accept `<url>` wrongly link here.

### What did you expect to see?

`<url>` is not just `url()`; it also consists of `src()`, which is like `url()` except literal URLs must be quoted, allowing things like `src(var(--my-src))`.

### Do you have any supporting links, references, or citations?

https://drafts.csswg.org/css-values/#urls

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/url`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/url
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/url/index.md
* Last commit: https://github.com/mdn/content/commit/875d97804293021faaf66c16a76e2f31a6df56e1
* Document last modified: 2024-05-06T13:36:09.000Z

</details>

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- fix https://github.com/mdn/content/issues/35029

The `<url>` is not the same as `<url()>` anymore. The PR creates a new page for `<url>` and renames `url()` page to `url_function`. The `src()` hasn't been implemented by any browser yet so haven't created a page for it.

### CODE REVIEW

**Comment by Josh-Cena** [comment]
Based on `color_value` and `position_value`, I thought we would also have `url_value` and `url` instead of `url` and `url_function`?

**Comment by OnkarRuikar** [comment]
> Based on `color_value` and `position_value`, I thought we would also have `url_value` and `url` instead of `url` and `url_function`?

Ohh. I found `Web/CSS/fit-content` vs. `Web/CSS/fit-content_function` so I mimicked that.

Couple of things here:
- The `color_value` and `position_value` are not accurate because they are types and not literal values. The `color_type`, `position_type`, and `url_types` would be more accurate.
- The `url_function` is more descriptive than the plain `url`. Also, when people search for the term "URL" they should land on `<url>` page and not on `url()`.
- In the future, when we create `src()`, it should be named `src_function` because `src` is a fairly common term in HTML and APIs.

File: files/en-us/web/css/url_function/index.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+  <li>Three</li>
+</ul>
+```
+
+#### CSS
+
+```css
+li::after {
+  content: " - "
+    url("https://mdn.github.io/shared-assets/images/examples/star.gif");
```
Comment (OnkarRuikar) [inline files/en-us/web/css/url_function/index.md:150]:
```suggestion
    url("https://mdn.github.io/shared-assets/images/examples/star-white_32x32.png
");
```

File: files/en-us/web/css/url_function/index.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+  <li>Three</li>
+</ul>
+```
+
+#### CSS
+
+```css
+li::after {
+  content: " - "
+    url("https://mdn.github.io/shared-assets/images/examples/star-white_32x32.png
```
Comment (OnkarRuikar) [inline files/en-us/web/css/url_function/index.md:150]:
```suggestion
    url("https://mdn.github.io/shared-assets/images/examples/star-white_16x16.png
```

**Comment by OnkarRuikar** [comment]
As decided, now using the explicit names `url_value` and `url_function`.

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**